### ISSUE TITLE
Crash on start-up

### ISSUE THREAD

**Issue by beschoenen** [ccrama/Slide#2675]
Slide version: 5.8.1
Android version: 7.0

Just updated to the latest beta via the play store and slide crashes after loading any sub, I can use slide until it loads the sub. Can also access the settings fine when using the cog in the settings app.

Stacktrace:
```
03-03 22:22:33.662 15199 15235 E AndroidRuntime: FATAL EXCEPTION: AsyncTask #2
03-03 22:22:33.662 15199 15235 E AndroidRuntime: Process: me.ccrama.redditslide, PID: 15199
03-03 22:22:33.662 15199 15235 E AndroidRuntime: java.lang.RuntimeException: An error occurred while executing doInBackground()
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at android.os.AsyncTask$3.done(AsyncTask.java:318)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:243)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:761)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'com.fasterxml.jackson.databind.JsonNode com.fasterxml.jackson.databind.JsonNode.get(int)' on a null object reference
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at me.ccrama.redditslide.SubmissionCache.getCrosspostSpannable(SubmissionCache.java:99)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at me.ccrama.redditslide.SubmissionCache.getCrosspostLine(SubmissionCache.java:51)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at me.ccrama.redditslide.SubmissionCache.cacheInfo(SubmissionCache.java:63)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at me.ccrama.redditslide.SubmissionCache.cacheSubmissions(SubmissionCache.java:43)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at me.ccrama.redditslide.Adapters.SubredditPosts$LoadData.doInBackground(SubredditPosts.java:439)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at me.ccrama.redditslide.Adapters.SubredditPosts$LoadData.doInBackground(SubredditPosts.java:281)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at android.os.AsyncTask$2.call(AsyncTask.java:304)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
03-03 22:22:33.662 15199 15235 E AndroidRuntime: 	... 4 more
03-03 22:22:33.664  3272 14377 W ActivityManager:   Force finishing activity me.ccrama.redditslide/.Activities.MainActivity
```

**Comment by TiMbl4** [comment]
Same here. For giggles cleared cache and data.

Slide version: 5.8.1
Android version: 7.1.1

Yet, I'm not using the beta channel, I'm on stable. I've updated via the play store moments ago.

**Comment by zer0fl0w** [comment]
+1

app always fc's after latest update from just an hour ago...


 
Slide version: 5.8.1

Android version: 8.0.0

**Comment by zer0fl0w** [comment]
fixed with v5.8.2

thanks for the instant help.

[closed]

**Comment by beschoenen** [comment]
Confirmed fixed in 5.8.2
